button-examples/Web/web-button-example.md

```diff
@@ -43,6 +42,8 @@ Install the `mdc-button` or `mdc-icon-button` component before including it in y
 npm install @material/buttons
 ```
 
+// It would help to have an image differentiating button an icon button since we don't show icon button or discuss it yet in this section
+
```

Comment:
I hemmed about this a great deal: I decided that all installation/setup instructions should be at the top of the document to limit repetition to avoid making this feel like two separate docs. Ultimately, I thought this doc should mirror the design article to present the canonical types as part of a single component.

@barbaraeldredge, @ajsecord  Thoughts?

Reply:
Reading through this, I found it really confusing because you mention four button types, then list two files to install, with no mapping between design names and the MDC component names. At this point in the doc, I have no idea whether I need mdc-button or mdc-icon-button.

This could be as simple as a single sentence after the diagram on L24 (or else in the intro to the install section).

Text, Outlined, and Contained buttons are implemented using the `mdc-button` component. Toggle buttons are implemented using the `mdc-icon-button` component. (I _think_ that's what I understand from the docs, but I could be missing something.)

Reply:
I can add the sentence to my branch, but I can't seem to directly edit this PR.